test/unit/steps/representative/representative-details/RepresentativeDetails.test.js

```diff
@@ -43,15 +45,44 @@ describe('RepresentativeDetails.js', () => {
 
     });
 
+    describe('get getCYAName()', () => {
+
+        beforeEach(() => {
+            representativeDetails.fields.nameOrganisation.firstName.value = '';
+            representativeDetails.fields.nameOrganisation.lastName.value = '';
+        });
+
+        it('should return Not Provided if firstName or lastName has not been set', () => {
+            expect(representativeDetails.CYAName).to.equal(userAnswer.NOT_PROVIDED);
+        });
+
+        it('should return the firstName if only the firstName has been set', () => {
+            representativeDetails.fields.nameOrganisation.firstName.value = 'FirstName';
+            expect(representativeDetails.CYAName).to.equal('FirstName');
+        });
+
+        it('should return the lastName if only the lastName has been set', () => {
+            representativeDetails.fields.nameOrganisation.lastName.value = 'LastName';
+            expect(representativeDetails.CYAName).to.equal('LastName');
+        });
+
+        it('should return the full name if both firstName and lastName has been set', () => {
+            representativeDetails.fields.nameOrganisation.firstName.value = 'FirstName';
+            representativeDetails.fields.nameOrganisation.lastName.value = 'LastName';
+            expect(representativeDetails.CYAName).to.equal('FirstName LastName');
```

Comment:
Pad out both `FirstName` and `LastName` both sides to ensure the `trim()` is working as we expect.